packages/core/src/analytics/integrations/Omnitracking/definitions.js

```diff
@@ -551,6 +551,24 @@ export const trackEventsMapper = {
     tid: 2927,
     loginType: data.properties?.method,
   }),
+  [eventTypes.INTERACT_CONTENT]: data => {
+    const properties = data.properties;
+    if (!properties?.contentType && !properties?.interactionType) {
+      logger.warn(
+        `[Omnitracking] - Event ${data.event} requires at lease properties contentType and interactionType
```

Comment:
`[Omnitracking] - Event ${data.event} requires at **least** the properties "contentType" and "interactionType"`

Also just a nitpick: add a blank line between the const and the if (line 555)

Reply:
I spotted this typo and this break line on next task, then I resolved in next PR.

But this branch will need rebase, so I will fix this, before the merge.